main.py

```diff
@@ -0,0 +1,116 @@
+#Amazon data set VADER analysis
+
+import pandas as pd
+import numpy as np
+import matplotlib.pyplot as plt
+import seaborn as sns
+import nltk
+from nltk.sentiment import SentimentIntensityAnalyzer
+from tqdm.notebook import tqdm
+
+#nltk.download('punkt')
+#nltk.download('averaged_perceptron_tagger')
+#nltk.download('maxent_ne_chunker')
+#nltk.download('words')
+#nltk.download('vader_lexicon')
+
+plt.style.use('ggplot')#
+
+
+#reading data and basic analysis
+data = pd.read_csv('./data/Reviews.csv')
+data = data.head(1000) #using 1000 rows
+#print(data.shape)
+
+ax = data['Score'].value_counts().sort_index().plot(kind = 'bar',title = 'Count of Reviews',\
+                                               figsize = (10, 5))
+ax.set_xlabel('Review Stars')
+plt.show()
+
+polarity = []
+sia = SentimentIntensityAnalyzer()
+
+for i in data['Id']:
+    tmp = []
+    tmp.append(i)
+    polarity.append(tmp)
+q = 0
+for i in data['Text']:
+    polarity[q].append(sia.polarity_scores(i))
+    q += 1

```

Comment:
Variables should be descriptive. 
- `i` is typically used for numbers, indexes etc, but using it for text is confusing. 
- using `q` for an index is also confusing, actually here `i` would be better

If you need an index in iteration, use `enumerate`:

```suggestion
for idx, text in enuerate(data['Text']):
    polarity[q].append(sia.polarity_scores(i))
```

Reply:
I get rid of loops, fixed